src/components/Card/Card.jsx

```diff
@@ -4,49 +4,58 @@ import styled from "styled-components";
 import data from "../../data.json";
 
 const CardStyled = styled.div`
-  perspective: 1000px;
-  padding-top: 0;
-
   .card {
     position: relative;
+  }
+
+  .card .front,
+  .card .back {
```

Comment:
Scoped bir css yazdığımız için buradaki `front` ve `back` sınıfları sadece bu bileşenin içindekiler ile eşleşecek o yüzden ayrıca `card` ın altındaki `front` vs. dememize gerek yok. Aşağıdaki gibi yapmak yeterli gibi görünüyor.

```css
.front,
.back {
 /* ... */
}
``` 

Ayrıca isimlendirme için şu yöntemi de kullanabilirsin. `card-front` `card-back`